parser/parse.go

```diff
@@ -443,7 +443,11 @@ func (b *Builder) typeCheckPackage(pkgPath importPathString) (*tc.Package, error
 		// method. So there can't be cycles in the import graph.
 		Importer: importAdapter{b},
 		Error: func(err error) {
-			klog.V(2).Infof("type checker: %v\n", err)
+			if logErr {
+				klog.V(2).Infof("type checker: %v\n", err)
+			} else {
+				klog.V(3).Infof("type checker: %v\n", err)
+			}
```

Comment:
The name `logErr` isn't really accurate here because we end up logging errors for packages with `logErr` at v=3. I do think we should allow users to view these errors if they want so don't want to remove them entirely.

Open to suggestions for better name if this approach makes sense.